.github/ISSUE_TEMPLATE/dabs-issue.md

```diff
@@ -0,0 +1,33 @@
+---
+name: DABs Issue
```

Comment:
I.e. "Bug report for Databricks Asset Bundles"

Reply:
done